addons/binding/org.openhab.binding.vallox/README.md

```diff
@@ -0,0 +1,52 @@
+# openhab-vallox
+Interface to a vallox central venting unit.
+
+Tested with 
+* Vallox KWL 90 SE and 
+* USR-TCP232-410 RS485-to-ethernet interface
+
+## Prerequisite
+
+This Openhab2 bundle so far only works with a RS485-to-ethernet gateway.
+It does not support a local serial interface of the device executing OH2.
+
+I do not have any experience with direct serial RS485 communication under Java and
+do not have any device for testing it. So if you can contribute any code for direct
+RS485 serial write and read, I might easily merge that.
+
+## Configuration
+
+Configure the serial interface such that it acts as TCP server on some port of your choice. 
+Configure the serial side as 
+* baudrate: 9600
+* data size: 8 bit
+* parity: none
+* stop bits: 1 
+
+## Install
+
+1. open paper ui in browser
+2. Goto Configuration -> System and disable Item Linking Simple mode (so no items will be created automatically)
+3. check that Configuration -> Bindings lists the Vallox binding
+4. go to Configuration -> Things
+5. add Vallox KWL Serial Binding Thing (exactly one)
+6. enter configuration parameters (host and port of serial gateway)
+7. The thing should show status `ONLINE`; if not, go into the details of the thing. The status there should include some error message.
+8. make sure you create items for all the many channels of the binding
+  * you can use [demo.items](conf/demo.items) to copy/paste the available items to your own item file
```

Comment:
I wonder if it gonna work on [auto-generated docs page](http://docs.openhab.org/addons/bindings.html). Maybe it'd be a good idea to include all example Items, Sitemaps, Rules etc. in this file and maybe create a Table of Content?
```
### vallox.things
...

### vallox.items
Number FanSpeed (Vallox) { channel="..." }

### vallox.sitemap
...
```